src/main/resources/translations.json

```
@@ -1,7 +1,51 @@
 {
   "de-DE": [
     {
-
+      "test": "Dies ist ein test %s",
+      "arenaIsFull": "Die Arena ist gerade besetzt, du wurdest in die Warteliste aufgenommen",
+      "addedToPlayerList": "Du wurdest der Liste hinzugefügt und hast eine Difficulty von: ",
+      "playerNotOP": "Du bist nicht dazu berechtigt",
+      "changedDiff1": "Deine Schwierigkeit wurde um 1 auf ",
```

String templates should be used
```suggestion
      "changedDiff": "Deine Schwierigkeit wurde um 1 auf %s erhöht",
```

In the class the used strings can be simply added behind a comma. (→ See main plugin)